packages/clay-color-picker/package.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
We can keep the information pattern in `package.json` equal to what we did in the other components. See [package.json from ClayDatePicker](https://github.com/liferay/clay/blob/9630c7157d9cb0b8c270a26353ceb630f0045c08/packages/clay-date-picker/package.json)